### DISCUSSION TITLE
How to generate link to section/index.html

### DISCUSSION THREAD

**Discussion by justf0rfun** [mkdocs/mkdocs#2392] [Q&A]
Hello. 

I want to use mkdocs to generate a site that I use only locally without a web server. 

Unfortunately AFAICS mkdocs relies on the web server to automatically find section/index.html on sections respective directories, where the generated links end with "section/". That does not work without a web server. 

For me it would be enough if the generated links would go directly to section/index.html.

Is that possible with mkdocs and if yes: how do I do that. Thx in advance.

**Comment by waylan** [comment]
You need to set the [use_directory_urls][1] setting to `False`.

[1]: https://www.mkdocs.org/user-guide/configuration/#use_directory_urls

**Comment by justf0rfun** [comment]
Thanks. Somehow I missed that.

It works except one link AFAICS: The link to the home or main site goes to site/ instead of site/index.html (or whatever has been configured to be the start page file). 

I could file a bug if who want me to.

Thanks again.

**Comment by mstephenson6** [comment]
I had this behavior as well. I get the links to html files from `use_directory_urls` only when I put `*.md` values in my `nav` config. Omitting `.md` from the `nav` entries seems to cause mkdocs to ignore `use_directory_urls`.

**Comment by huifeideteji** [comment]
well,if the setting `use_directory_urls` doesn't work,please make sure u have executed `mkdocs build`,and hard refresh ur browser.i thik it's caused by browser cache.

**Reply by huifeideteji** [reply]
@justf0rfun @mstephenson6
